pkg/alerts/nrql_conditions.go

```diff
@@ -6,6 +6,40 @@ import (
 	"github.com/newrelic/newrelic-client-go/pkg/errors"
 )
 
+// AlertsNrqlConditionExpiration - **Preview access:** These fields may be viewed and set, but will not be active until the release date.
+// Settings for how violations are opened or closed when a signal expires.
+type AlertsNrqlConditionExpiration struct {
+	CloseViolationsOnExpiration bool `json:"closeViolationsOnExpiration"`
+	ExpirationDuration          int  `json:"expirationDuration"`
+	OpenViolationOnExpiration   bool `json:"openViolationOnExpiration"`
+}
+
+// AlertsNrqlConditionSignal - Configuration that defines the signal that the NRQL condition will use to evaluate.
+type AlertsNrqlConditionSignal struct {
+	EvaluationOffset int              `json:"evaluationOffset"`
+	FillOption       AlertsFillOption `json:"fillOption"`
+	FillValue        float64          `json:"fillValue"`
```

Comment:
Fill value is nullable in the API, suggest making it a pointer

Reply:
roger that